### PR TITLE
Remove `DDS` support

### PR DESCRIPTION
**Pull request by caiotoledo**
Waiting for decision if DDS example should be removed.

### CODE REVIEW

**Comment by pablogh-2000** [comment]
> Waiting for decision if DDS example should be removed.

Please go ahead.

**Review by pablogh-2000 — COMMENTED** [review]
Aren't there also some chapters dedicate to DDS that should be removed?

File: CMakeLists.txt [hunk 1]
```diff
@@ -68,12 +68,7 @@ SET(BUILD_SHARED_LIBS "ON" FORCE)
 find_package(Klepsydra 8 REQUIRED)
 
 include(KpsrCodeGen)
-# Generate header files for messages/yaml files before compiling any code
-set(DDS_CODEGEN "True")
-if(KPSR_WITH_DDS)
-set(DDS_CODEGEN "False")
-endif()
-KpsrEventGenerator("${CMAKE_CURRENT_SOURCE_DIR}/kidl" "${CMAKE_CURRENT_SOURCE_DIR}/gen" "klepsydra/tutorial" "False" "True" "${DDS_CODEGEN}")
+KpsrEventGenerator("${CMAKE_CURRENT_SOURCE_DIR}/kidl" "${CMAKE_CURRENT_SOURCE_DIR}/gen" "klepsydra/tutorial" "False" "True" "False")
```
Comment (mharshe) [inline CMakeLists.txt:71]: the option is `disableDDS = ?` so you have to set it to `True` to disable it. A little counterintuitive :)

**Comment by mharshe** [comment]
Any idea why the "Build Project" section of the CI debug run takes 0ms to run? https://github.com/klepsydra-technologies/kpsr-tutorial/actions/runs/5316095037/jobs/9625247128?pr=40

File: CMakeLists.txt (hunk 1 above)
Comment (caiotoledo) [reply CMakeLists.txt]: Indeed, my bad I didn't notice :)
Done in 95894e7fe7a716c0b76da18ee2025c38ae3f029f.

File: tutorials/chapter2.md [hunk 2]
```diff
@@ -443,4 +378,4 @@ The subscriber factory in the ZMQ case is always the ```kpsr::zmq_mdlw::FromZmqM
 <a name="zmq-env"></a>
 ### ZMQ Env
 
-The ```kpsr::zmq_mdlw::ZMQEnv``` is similar to the ```kpsr::dds_mdlw::DDSEnv``` in that it is a wrapper to the ```kpsr::YamlEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as DDS or ROS.
+```kpsr::zmq_mdlw::ZMQEnv``` is a wrapper to the ```kpsr::YamlEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as ROS.
```
Comment (mharshe) [inline tutorials/chapter2.md:381]:
```suggestion
```kpsr::zmq_mdlw::ZMQEnv``` is a wrapper to the JSON based ```kpsr::ConfigurationEnvironment``` class that holds the configuration for the environment (or global data parameters to be shared over the ZMQ network), along with an ability to share this data over the network. It functions similarly, allowing reading global parameters and writing to them too. The changes to the parameters are visible immediately to all Klepsydra applications that are on the ZMQ network. As it inherits from the ```kpsr::Environment``` class, the interface is the same as ROS.
```
This is unrelated to DDS, but a necessary fix.

**Review by mharshe — COMMENTED** [review]
Minor fix. You can squash the commits now.

File: tutorials/chapter2.md (hunk 2 above)
Comment (caiotoledo) [reply tutorials/chapter2.md]: Done.

**Review by pablogh-2000 — APPROVED** [review]

**Pull request merged**